### PR TITLE
Create 2017-05-02-textworkshop17.md

### PR DESCRIPTION
**Pull request by kbenoit**
Add the blog post requested by @stefaniebutland about http://textworkshop17.ropensci.org.

### CODE REVIEW

File: _posts/2017-05-02-textworkshop17.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+layout: post_discourse
+title: Text Analysis R Developers' Workshop 2017
+date: 2017-05-03
+authors:
+  - name: Ken Benoit
+    url: http://kenbenoit.net
+tags:
+- R
+- text analysis
+---
```
Comment (stefaniebutland) [inline _posts/2017-05-02-textworkshop17.md]: please add tags:
- community
- meetings

File: _posts/2017-05-02-textworkshop17.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
...t brought together developers of R packages for working with text and text-related data.  This included a wide range of applications, including string handling ([**stringi**](https://github.com/gagolews/stringi)) and tokenization (the rOpenSci-onboarded [**tokenizers**](https://github.com/ropensci/tokenizers), [**KoNLP**](https://github.com/haven-jeon/KoNLP)), corpus and text processing ([**readtext**](https://github.com/kbenoit/readtext), [**tm**](http://tm.r-forge.r-project.org), [**quanteda**](http://quanteda.io), and [**qdap**](https://github.com/trinker/qdap)), natural language processing (NLP) such as part of speech and dependency tagging ([**cleanNLP**](https://github.com/statsmaths/cleanNLP), [**spacyr**](https://github.com/kbenoit/spacyr)), and the statistical analysis of textual data ([**stm**](http://www.structuraltopicmodel.com), [**text2vec**](https://github.com/dselivanov/text2vec), and [**koRpus**](https://reaktanz.de/?c=hacking&s=koRpus)) -- although this list is hardly complete.  The main objective was to bring together experts working on various aspects of text processing and text analysis using R, to discuss common challenges and identify collaborative solutions.
```
Comment (stefaniebutland) [inline _posts/2017-05-02-textworkshop17.md]: consider linking to workshop site so people can see the participants http://textworkshop17.ropensci.org/

File: _posts/2017-05-02-textworkshop17.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+The workshop was semi-structured, somewhere in between a traditional agenda-driven workshop and an "[unconference](http://unconf17.ropensci.org)".  On Day 1 and in the first session of Day 2, we held a set of seven topically focused roundtables, with between 3-4 participants plus an appointed moderator.  Organized around themes or problems, such as segmenting and tokenizing text, or trying to parse linguistic structure and parts of speech, each session lasted about an hour and features 3-4 participants and a moderator. (The full list is [here](https://github.com/ropensci/textworkshop17/blob/master/preliminary_schedule.md#schedule).) This gave everyone a chance to participate in a structured event, and served to guide discussion.  This proved invaluable in identifying challenges, especially the final roundtable on Saturday morning to discuss package interoperability, which had been aided greatly by the overall discussion that took place before across the first six roundtables.  (A group dinner at a [local brewpub](http://www.templebrewhouse.com/brewery/) the evening before also helped spur this discussion.)
```
Comment (stefaniebutland) [inline _posts/2017-05-02-textworkshop17.md]: omit "...and features 3-4 participants and a moderator" right before "(The full list is [here]..." as it's a repeat of above

File: _posts/2017-05-02-textworkshop17.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (stefaniebutland) [inline _posts/2017-05-02-textworkshop17.md]: Thank you @kbenoit. This post is great. Captures intentions, richness of specific work done, as well as the spirit of rOpenSci community! I have only minor suggested edits and @sckott agrees we can post as soon as you incorporate these and give us the ok.

File: _posts/2017-05-02-textworkshop17.md (hunk 2 above)
Comment (stefaniebutland) [inline _posts/2017-05-02-textworkshop17.md]: please put pkg names in backticks, no boldface e.g. `tokenizers`
Will appear as in this post about `tabulizer`: https://ropensci.org/blog/blog/2017/04/18/tabulizer

**Comment by sckott** [comment]
thanks @kbenoit

**Comment by sckott** [comment]
@kbenoit assuming you're happy with it, and ready to post?

**Comment by stefaniebutland** [comment]
@kbenoit told me it's ready to post
Thanks @sckott

**Pull request merged**